src/components/signin/signin.js

```diff
@@ -13,7 +12,9 @@ const SignIn = () => {
           className="btn btn-red"
           onClick={() => {
             loginWithRedirect({
-              appState: { targetUrl: withPrefix(window.location.pathname) },
+              appState: {
+                targetUrl: window.location.pathname.split('/').pop(),
```

Comment:
can you move the `split('/').pop` to a separate line. I would recommend setting this as a variable above.
This can help in debugging to simplify long code lines that have multiple operations

Reply:
I created a new variable, but left `split('/').pop` on the same line to keep `yarn lint` happy